packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -365,11 +361,10 @@ export function isNetwork(chainId: ChainId) {
     isHolesky ||
     isArbitrumSepolia ||
     isCustomOrbitChain ||
-    isStylusTestnet ||
     isStylusTestnetV2 ||
     isTestnetOrbitChain
 
-  const isStylusTestnetV1orV2 = isStylusTestnet || isStylusTestnetV2
+  const isStylusTestnetV1orV2 = isStylusTestnetV2
```

Comment:
This is only used in `TransferPanelSummary`. We can replace `isStylusTestnetV1orV2` with `isStylusTestnetV2` there and remove this line.